pact/coin-contract/v2/verified/coin.pact

```diff
@@ -0,0 +1,590 @@
+(module coin GOVERNANCE
+
+  @doc "'coin' represents the Kadena Coin Contract. This contract provides both the \
+  \buy/redeem gas support in the form of 'fund-tx', as well as transfer,       \
+  \credit, debit, coinbase, account creation and query, as well as SPV burn    \
+  \create. To access the coin contract, you may use its fully-qualified name,  \
+  \or issue the '(use coin)' command in the body of a module declaration."
+
+  @model
+    [ (defproperty conserves-mass
+        (= (column-delta coin-table 'balance) 0.0))
+
+      (defproperty valid-account (account:string)
+        (and
+          (>= (length account) 3)
+          (<= (length account) 256)))
+
+      (defproperty enforce-custody (account:string)
+        (row-enforced coin-table 'guard account))
```

Comment:
Just to confirm, this adds an additional layer of security which reinforces that the tx sender is authorized to modify an account's row via `enforce-keyset`?  It looks like this property is enforced as part of the `gas-guard` and any debiting operations. This makes sense to add!

Reply:
Yes, it leverages `row-enforced` which is a really great property!